core/provisioning-java/src/main/java/org/apache/syncope/core/provisioning/java/propagation/AbstractPropagationTaskExecutor.java

```diff
@@ -383,6 +383,20 @@ protected Uid delete(
 
             connector.delete(objectClass, uid, null, propagationAttempted);
             result = uid;
+            // SYNCOPE-1809 remove uidOnCreate attribute, if any
+            taskInfo.getResource()
+                    .getProvisionByAnyType(taskInfo.getAnyType())
+                    .filter(provision -> provision.getUidOnCreate() != null)
+                    .ifPresent(provision -> {
+                        LOG.debug("Removing uidOnCreate [{}] attribute from [{}] on delete",
```

Comment:
either remove this logging statement or add it for the other case when `getUidOnCreate()` is used in this class

Reply:
Added logging also in the other section.